README.md

```diff
@@ -26,26 +26,16 @@ This package will launch a `purescript-language-server` process, which will auto
 
 Multi-root workspaces should be supported via a multiple language server approach.
 
-For all functions provided by the IDE server you will need to build your project first!
+For all functions provided by the IDE server you will need to build your project first! A rebuild is automatically triggered when saving a `.purs` file.
```

Comment:
Not sure of the intention of this change, I think it confuses the issue - I think the point about building first is that otherwise "nothing" will work - rebuild is a nice feature, but that's rather separate from the build config. But then the "the package will start on opening a .purs file" is similarly not quite in the right place imo

Think that if we mention the .purs file triggering extension / saving and then go into the "build your project, it would make most sense"